src/main/java/at/hannibal2/skyhanni/utils/TimeLimitedCache.kt

```diff
@@ -21,7 +21,16 @@ class TimeLimitedCache<K: Any, V: Any>(
 
     fun getOrNull(key: K): V? = cache.getIfPresent(key)
 
-    fun getOrPut(key: K, defaultValue: () -> V) = getOrNull(key) ?: defaultValue().also { set(key, it) }
+    fun getOrPut(key: K, defaultValue: () -> V): V {
```

Comment:
Why do we change this clean kotlin logic into longer java logic?

Reply:
I think a wrong merge here